test/check-machines-settings

```diff
@@ -215,9 +215,15 @@ class TestMachinesSettings(VirtualMachinesCase):
         self.goToVmPage("subVmTest1")
         b.wait_in_text("#vm-subVmTest1-state", "Running")
 
+        cpu_model = "host-passthrough"
+        # virt-install changed the default to "host-passthrough"
+        # https://github.com/virt-manager/virt-manager/commit/2c477f330244e04614e174f50fbf37260c535705
+        if m.image in ["fedora-36"]:
+            cpu_model = "host-model"
```

Comment:
Let's invert that list, so that it shrinks over time instead of grows. We'll have to adjust it anyway when debian-testing, c9s etc. pick up the new version.